src/y-labels.js

```diff
@@ -22,7 +22,7 @@ export default class YLabels extends React.Component {
     }
 
     renderLabels() {
-        const day = moment();
+        const day = moment().startOf('hour');
```

Comment:
There is a not so nice hack for this, setting the `timeFormat` to `HH:00`. But I prefer your solution